Flow.Launcher/SettingWindow.xaml

```diff
@@ -32,70 +33,72 @@
 
     <TabControl Height="auto" SelectedIndex="0">
         <TabItem Header="{DynamicResource general}">
-            <StackPanel Orientation="Vertical">
-                <CheckBox Margin="10" IsChecked="{Binding PortableMode}">
-                    <TextBlock Text="Portable Mode" />
-                </CheckBox>
-                <CheckBox Margin="10" IsChecked="{Binding Settings.StartFlowLauncherOnSystemStartup}"
+            <ScrollViewer ui:ScrollViewerHelper.AutoHideScrollBars="True" Margin="60,30,0,30">
+                <StackPanel Orientation="Vertical">
+                    <ui:ToggleSwitch Margin="10" IsOn="{Binding PortableMode}">
+                        <TextBlock Text="Portable Mode" />
```

Comment:
small nitpick that I spotted after approving: this should be a resource too

Reply:
Good pick up. I will make an issue and fix tonight. I am not able to right now.